src/test/java/com/jfrog/ide/common/yarn/YarnDriverTest.java

```diff
@@ -0,0 +1,91 @@
+package com.jfrog.ide.common.yarn;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.apache.commons.io.FileUtils;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.testng.Assert.*;
+
+/**
+ * Test the 'why' functionality in YarnDriver.
```

Comment:
Why name the class "YarnDriverTest" if it is intended to check only the "why" functionality? :)